### ISSUE TITLE
Clicktag

### ISSUE THREAD

**Issue by fleurdeboer** [airbnb/lottie-web#2837]
Hi! I used the amazing plug-in to export banners. Used the Banner setting, but now I got feedback from my client/campaign operators that "clicktag" doesn't work (also checked with the validator and got this message). So they can't attach an URL to it. Is there a solution for this? Thanks, big help!!

<img width="848" alt="clicktag" src="https://user-images.githubusercontent.com/111369271/184910608-639ebc7c-2259-407d-8635-258d6c744e1c.png">
!

**Comment by a-mackenzie56** [comment]
I'm having the same issue, and I can't seem to find an obvious fix. Is this because of an update to Google's ad platform? Because I've never had any issues in the past.

**Comment by bodymovin** [comment]
yes, it seems Google ad platform is displaying an alert for xmlHttpRequest of the 'GET' type.
You can try doing a search and replace of `'GET'` with `['G','E','T'].join('')` in the exported file and it should fix it.
